RoeiVerenigingLibary/Boat.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace RoeiVerenigingLibary
+{
+    public class Boat
+    {
+        public int Id { get; set; }
+        public Boolean CaptainSeat { get; set; }
+        public int Seats { get; set; }
+        //public Url Image { get; set; }
```

Comment:
deze regel zit niet in database moet verwijderd worden. komt apparte tabel voor met afbeeldingen